README.md

```diff
@@ -1,2 +1,168 @@
-# lite
-A single container version of Appwrite with minimum must have features.
+<p align="center">
+    <a href="https://appwrite.io" target="_blank"><img width="260" height="39" src="https://appwrite.io/images/github-logo.png" alt="Appwrite Logo"></a>
+    <br />
+    <br />
+    <b>A complete backend solution for your [Flutter / Vue / Angular / React / iOS / Android / *ANY OTHER*] app</b>
+    <br />
+    <br />
+</p>
+
+
+[![Discord](https://img.shields.io/discord/564160730845151244?label=discord&style=flat-square)](https://appwrite.io/discord)
+[![Docker Pulls](https://img.shields.io/docker/pulls/appwrite/appwrite?color=f02e65&style=flat-square)](https://hub.docker.com/r/appwrite/appwrite)
+[![Build Status](https://img.shields.io/travis/com/appwrite/appwrite?style=flat-square)](https://travis-ci.com/appwrite/appwrite)
+[![Twitter Account](https://img.shields.io/twitter/follow/appwrite_io?color=00acee&label=twitter&style=flat-square)](https://twitter.com/appwrite_io)
+[![Follow Appwrite on StackShare](https://img.shields.io/badge/follow%20on-stackshare-blue?style=flat-square)](https://stackshare.io/appwrite)
+
+Appwrite lite is the stripped down, single container version of Appwrite, with non essential services removed. Optimized for low resource systems.
+
+
+Table of Contents:
+
+- [Whats different](#whats-different)
+- [Installation](#installation)
+- [Getting Started](#getting-started)
+    
+## What's different
+Appwrite lite is simpler and light weight version. So we have removed few functionalities and made some fundamental changes as follows.
+1. All the services are running inside a single container using supervisord.
+2. Services like ClamAV antivirus and InfluxDB and Telegraf for usage stats have been removed.
+
+In case of features, all the features of Appwrite is present except for usage stats and Antivirus.
```

Comment:
Maybe we should add a section: `Who is it good for?` also explaining why use this over regular version. I would also mention on the the `What's different` version, which services are omitted, also explaining why each one was omitted.